dev/com.ibm.ws.jpa_22_fat/fat/src/com/ibm/ws/jpa/JPAAppClientTest.java

```diff
@@ -0,0 +1,222 @@
+/*******************************************************************************
+ * Copyright (c) 2017 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package com.ibm.ws.jpa;
+
+import java.io.File;
+import java.util.List;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.config.Application;
+import com.ibm.websphere.simplicity.config.ClientConfiguration;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyClient;
+import componenttest.topology.impl.LibertyClientFactory;
+
+@RunWith(FATRunner.class)
+public class JPAAppClientTest {
+    private static final String APPCLIENT_ROOT = "test-applications/appclient";
+    private final static String CLIENT_NAME = "JPAAppClient";
+    private final static String PKG_ROOT = "jpaappcli";
+
+    protected static LibertyClient client = LibertyClientFactory.getLibertyClient(CLIENT_NAME);
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+
```

Comment:
Do we need these methods even though they are empty?

Reply:
We probably don't, I just left them in place should I change my mind later and add content.  I don't think it hurts anything.